### ISSUE TITLE
should be able to use batong / shocked grille as defib

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#21472]
## Description
with increased damage and less likely to succeed / less airloss healing idk

reason: funny, actually useful for sec to use batong to maybe quickly defib someone during a fight

**Comment by Veritable-Calamity** [comment]
I like this idea as a ghetto medical way of doing things. That said, whomever undertakes this: please be careful with the values so that ghettomed doesn't become the go-to versus normal medical - people in medbay don't get enough respect as-is sometimes.

Also, when #19383 is finally in a finished state, might be worth having ghetto defibrillators cause damage to the heart or something.

**Comment by Jezithyr** [comment]
Yeah, I think that's a good idea! Tbh I was also thinking that enough electrical damage could act like a ghetto defib, so shocking someone with a MV cable or taser could restart their heart. (That's how defibs work IRL, they stop your heart so that it can get restarted with the proper rhythm, and HV electrical shocks do the same thing just with more nerve damage :P)

**Comment by beck-thompson** [comment]
haha love this idea!
